swap/src/config.rs

```diff
@@ -16,9 +16,23 @@ pub struct Config {
     pub monero_network: monero::Network,
 }
 
-impl Config {
-    pub fn mainnet() -> Self {
-        Self {
+// TODO: This trait is not needed
```

Comment:
Is this TODO correct? The test configuration seems to depend on the trait, e.g.:

```

impl GetConfig for SlowCancelConfig {
    fn get_config() -> Config {
        Config {
            bitcoin_cancel_timelock: Timelock::new(180),
            ..config::Regtest::get_config()
        }
    }
}
```

Reply:
Indeed, missed that.